File: 프로그래머스/lv1/42748. K번째수/K번째수.cs
using System;
using System.Collections.Generic; 

public class Solution {
   public List<int> solution(int[] array, int[,] commands)
            {
                List<int> answer = new List<int>();
                
                for (int i = 0; i<commands.GetLength(0); i++)
                {
                    List<int> ints = new List<int>();

                    int target = commands[i,2];
                    
                    for(int j = commands[i,0]; j <=commands[i,1]; j++)
                    {
                        ints.Add(array[j-1]); 
                    }

                    ints.Sort();
                    answer.Add(ints[target-1]);
                }

                return answer;
            }
}

File: 프로그래머스/lv1/42748. K번째수/K번째수.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

vector<int> solution(vector<int> array, vector<vector<int>> commands) {
    
    vector<int> answer;
    for(int i = 0; i<commands.size(); i++)
  {
    vector<int> test; 
        
    int target = commands[i][2];
        
    for(int j = commands[i][0]; j<=commands[i][1]; j++)
        test.push_back(array[j-1]);
     
    sort(test.begin(),test.end());
      
     answer.push_back(test[target-1]);
        
        
  }
    return answer;
}
  